main.go

```diff
@@ -1,14 +1,13 @@
 package main
 
 import (
-	"fmt"
 	"os"
 
 	"github.com/flatcar-linux/fleetlock/cmd"
 )
 
 func main() {
-	if err := cmd.Root.Execute(); err != nil {
-		fmt.Fprintf(os.Stderr, "unable to execute command: %v", err)
+	if err := cmd.Command().Execute(); err != nil {
+		os.Exit(1)
```

Comment:
No need to print the `err` ? it's already handled by the `Execute` ?

Reply:
Yep:
```console
$ ./fleetlock recursive-lock
Error: building the client: parsing URL: parse "": empty url
Usage:
  fleetlockctl recursive-lock [flags]

Flags:
  -h, --help   help for recursive-lock

Global Flags:
  -g, --group string   FleetLock group (default "default")
  -i, --id string      FleetLock instance ID (e.g. content of /etc/machine-id file)
  -u, --url string     FleetLock endpoint URL

```